<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="table">
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>60</number>
      </attribute>
      <column>
       <property name="text">
        <string>Date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>IRAP Hours</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Task Number</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Description</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="month_cbox">
         <item>
          <property name="text">
           <string>January</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>February</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>March</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>April</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>May</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>June</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>July</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>August</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>September</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>October</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>November</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>December</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="year_cbox"/>
       </item>
       <item>
        <widget class="QPushButton" name="update_table_btn">
         <property name="text">
          <string>Update Table</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="generate_files_btn">
         <property name="text">
          <string>Generate Files</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionEmployee_Name"/>
    <addaction name="actionGoogle_Sheets_ID"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionGoogle_Sheets_ID">
   <property name="text">
    <string>Google Sheets ID</string>
   </property>
  </action>
  <action name="actionEmployee_Name">
   <property name="text">
    <string>Employee Name</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
